src/main/java/seedu/ezmanager/commands/member/TeamMemberAssignToTaskCommand.java

```diff
@@ -26,19 +29,32 @@ public TeamMemberAssignToTaskCommand(HashMap<String, String> params, int project
         this.parse();
     }
 
+    /**
+     * Parse user parameter inputs for execution.
+     *
+     * @throws EzExceptions Invalid index when parameter values entered is not an integer.
+     */
     public void parse() throws EzExceptions {
         try {
             taskIndex = Integer.parseInt(getHashValue(params, "t")) - 1;
-        } catch (NumberFormatException e) {
-            throw new EzExceptions("invalidTaskID");
-        }
-        try {
             memberIndex = Integer.parseInt(getHashValue(params, "m")) - 1;
         } catch (NumberFormatException e) {
-            throw new EzExceptions("invalidTeamMemberID");
+            throw new EzExceptions("indexNonInteger");
         }
     }
 
+    /**
+     * Assign specified member to specified task.
```

Comment:
should we follow the way i suggested above?

Reply:
done!!